brangelina.vim

```diff
@@ -168,51 +149,29 @@ augroup customCommands
   nmap <silent> <localleader>a :TestSuite<CR>
   nmap <silent> <localleader>l :TestLast<CR>
   nmap <silent> <localleader>g :TestVisit<CR>
-  autocmd VimEnter *
-  \ command! -bang -nargs=* Ag
-  \ call fzf#vim#ag(<q-args>, '', { 'options': '--bind ctrl-a:select-all,ctrl-d:deselect-all' }, <bang>0)
   autocmd BufWritePre * Neoformat
 augroup END
 
 " # Commands
 " reload .config/nvim/init.vim
 command! ReloadConfig execute "source ~/.config/nvim/init.vim"
-" close hidden buffers
-command! -nargs=* Only call CloseHiddenBuffers()
-function! CloseHiddenBuffers()
-    " figure out which buffers are visible in any tab
-    let l:visible = {}
-    for l:t in range(1, tabpagenr('$'))
-        for l:b in tabpagebuflist(l:t)
-            let l:visible[l:b] = 1
-        endfor
-    endfor
-    " close any buffer that are loaded and not visible
-    let l:tally = 0
-    for l:b in range(1, bufnr('$'))
-        if bufloaded(l:b) && !has_key(l:visible, l:b)
-            let l:tally += 1
-            exe 'bw ' . l:b
-        endif
-    endfor
-    echon 'Deleted ' . l:tally . ' buffers'
-endfun
 
 command! -bang -nargs=* Rg
   \ call fzf#vim#grep(
   \   'rg --column --line-number --no-heading --color=always '.shellescape(<q-args>).'| tr -d "\017"', 1,
-  \   fzf#vim#with_preview({ 'options': '--bind ctrl-a:select-all,ctrl-d:deselect-all,ctrl-q:toggle-preview' },'up:60%'),
+  \   { 'options': '--bind ctrl-a:select-all,ctrl-d:deselect-all' },
   \   <bang>0)
 
 command! -bang -nargs=? -complete=dir Files
-  \ call fzf#vim#files(<q-args>, fzf#vim#with_preview({ 'options': '--bind ctrl-a:select-all,ctrl-d:deselect-all,ctrl-q:toggle-preview' }, 'up:60%'), <bang>0)
+  \ call fzf#vim#files(<q-args>, { 'options': '--bind ctrl-a:select-all,ctrl-d:deselect-all' }, <bang>0)
 
 " # Plugins
 function! BrangelinaPlugins()
   Plug 'Shougo/deoplete.nvim'
```

Comment:
deoplete breaks every month or so for me would love to have a better plugin

Reply:
Yeah. I'm also often confronted with a string of error messages everytime I close vim.